go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/giantswarm/k8sclient v0.2.0
 	github.com/giantswarm/k8sclient/v2 v2.0.0
 	github.com/giantswarm/k8sclient/v3 v3.1.0
-	github.com/giantswarm/k8scloudconfig/v6 v6.1.1
+	github.com/giantswarm/k8scloudconfig/v6 v6.2.4-0.20200612095134-8de112416492
```

Comment:
I'll make a release once this is tested to be good to go and then update this one.